### ISSUE TITLE
Some request examples dont seem to work

### ISSUE THREAD

**Issue by IdontKnowWhatToNameThisAccount** [woctezuma/download-steam-reviews#16]
I tried to do the 
`Download reviews for one appID, with specific request parameters (language, sentiment, store)`
 request example without any editing from the readme, but I always get the error
```
Traceback (most recent call last):
  File "C:\Users\samsample\Desktop\steamreviewdownload.py", line 11, in <module>
    review_dict, query_count = steamreviews.download_reviews_for_app_id(app_id,
  File "C:\Users\samsample\AppData\Roaming\Python\Python39\site-packages\steamreviews\download_reviews.py", line 296, in download_reviews_for_app_id
    num_reviews = query_summary['total_reviews']
KeyError: 'total_reviews'
```
I also tried to fix the weird line break in line 11/12 but that yields the same error

**Comment by woctezuma** [comment]
The error happens here:

https://github.com/woctezuma/download-steam-reviews/blob/4b8068fba3243500999864d34eb6b8a0ce3c2579/steamreviews/download_reviews.py#L296

which means `query_summary` does not have the field `total_reviews` after being returned here:

https://github.com/woctezuma/download-steam-reviews/blob/4b8068fba3243500999864d34eb6b8a0ce3c2579/steamreviews/download_reviews.py#L245

The issue would come from here:

https://github.com/woctezuma/download-steam-reviews/blob/4b8068fba3243500999864d34eb6b8a0ce3c2579/steamreviews/download_reviews.py#L194-L202

**Comment by woctezuma** [comment]
The error only happens with the filter by `review_type`.

```python
import steamreviews

app_id = 573170
review_dict, query_count = steamreviews.download_reviews_for_app_id(app_id, chosen_request_params={'review_type': 'positive'})
```

**Comment by woctezuma** [comment]
After some local tests:
```python
from download_reviews import download_reviews_for_app_id_with_offset

params = {}

app_id = 573170
_, _, query_summary, _, _ = download_reviews_for_app_id_with_offset(app_id,
                                                                    query_count=0,
                                                                    chosen_request_params=params)

print(query_summary)
```
returns:
```python
{'num_reviews': 100, 'review_score': 8, 'review_score_desc': 'Very Positive', 'total_positive': 342, 'total_negative': 15, 'total_reviews': 357}
```
whereas:
```python
from download_reviews import download_reviews_for_app_id_with_offset

params = {'review_type': 'positive'}

app_id = 573170
_, _, query_summary, _, _ = download_reviews_for_app_id_with_offset(app_id,
                                                                    query_count=0,
                                                                    chosen_request_params=params)

print(query_summary)
```
returns:
```python
{'num_reviews': 100}
```

**Comment by woctezuma** [comment]
This should be fixed in v0.9.5.
